shelley-ma/shelley-ma-test/cardano-ledger-shelley-ma-test.cabal

```diff
@@ -104,6 +104,7 @@ test-suite cardano-ledger-shelley-ma-test
       "-with-rtsopts=-K4m -M250m"
     build-depends:
       cardano-ledger-shelley-ma,
+      data-default-class,
```

Comment:
In one place we use `data-default`, in the other `data-default-class`. It looks like actually we should just use `data-default-class` everywhere.